Plugin~/WebRTCPlugin/GraphicsDevice/D3D12/D3D12GraphicsDevice.h

```diff
@@ -13,7 +13,7 @@ namespace webrtc = ::webrtc;
 
 #define DefPtr(_a) _COM_SMARTPTR_TYPEDEF(_a, __uuidof(_a))
 DefPtr(ID3D12CommandAllocator);
-DefPtr(ID3D12GraphicsCommandList4);
+DefPtr(ID3D12GraphicsCommandList);
```

Comment:
I think we should use the newer interface. What is the reason of this change?

Reply:
OK, I will revert this and test.

Reply:
Mmm, this changes is passed test on local PC but failed on Yamato.
Graphics driver on Bokken image is already updated. Do you have any idea?

Reply:
Hm, try getting the version of DX12 on both local PC and Yamato and make sure they are the same ?
https://support.microsoft.com/en-us/help/179113/how-to-install-the-latest-version-of-directx

Reply:
I got it!
The windows version is earlier than required version for test.
I am upgrading the Windows OS of the testing environment.